artiq/coredevice/coredevice_generic.schema.json

```
@@ -53,6 +53,10 @@
             "type": "boolean",
             "default": false
         },
+        "enable_acpki": {
+            "type": "boolean",
+            "default": false
```

That's only valid for Kasli-SoC and would be silently ignored otherwise. Would it make sense to include that check in the schema?

---

the jsonschema check does not allow additional properties. without this change the following error appears:
```
Traceback (most recent call last):
  File "/home/riesebos/artiq/lriesebos/artiq-zynq/src/gateware/kasli_soc.py", line 795, in <module>
    main()
    ~~~~^^
  File "/home/riesebos/artiq/lriesebos/artiq-zynq/src/gateware/kasli_soc.py", line 767, in main
    description = jsondesc.load(args.description)
  File "/nix/store/ifr9i5p6m3qw3swg2bgm4nkkrm8cl2y9-python3.13-artiq-9.9376+ee6bb2f.beta/lib/python3.13/site-packages/artiq/coredevice/jsondesc.py", line 33, in load
    validator.validate(result)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/nix/store/crmm5qwj1a7n5bw3q67zq03y661l7vax-python3-3.13.7-env/lib/python3.13/site-packages/jsonschema/validators.py", line 451, in validate
    raise error
jsonschema.exceptions.ValidationError: Additional properties are not allowed ('enable_acpki' was unexpected)
```

---

forced it to `false` on Kasli.